compare_predictions.py

```diff
@@ -64,10 +68,16 @@ def main():
 
     print(f'The best model is the one that generated {best_of_best.prediction} ({best_of_best.average_best_guess})')
 
+    # Save best model
     model_path = next((model for model in valohai.inputs('models').paths() if model_filename in model), '')
     if model_path:
         shutil.copy(model_path, valohai.outputs().path(model_filename))
 
+        # Save the metadata for the best model
+        metadata_path = valohai.outputs().path(model_filename+'.metadata.json')
```

Comment:
Lets do [string formatting](https://peps.python.org/pep-0498/) in one way. We already have f-strings in this file and repository, so the following would be more in line:

```python
metadata_path = valohai.outputs().path(f'{model_filename}.metadata.json')
```